src/kili/use_cases/project/project.py

```diff
@@ -28,18 +28,61 @@ def create_project(
         json_interface: Dict,
         title: str,
         description: str,
+        project_id: Optional[ProjectId],
         project_type: Optional[ProjectType],
         compliance_tags: Optional[ListOrTuple[ComplianceTag]],
     ) -> ProjectId:
-        """Create a project."""
-        project_id = self._kili_api_gateway.create_project(
-            input_type=input_type,
-            json_interface=json_interface,
-            title=title,
-            description=description,
-            project_type=project_type,
-            compliance_tags=compliance_tags,
-        )
+        """Create or copy a project if project_id is set."""
+        if project_id is not None:
+            project_copied = self._kili_api_gateway.get_project(
+                project_id=project_id, fields=["jsonInterface", "instructions"]
+            )
+            project_tag = self._kili_api_gateway.list_tags_by_project(
+                project_id=project_id, fields=["id"]
+            )
+            new_project_id = self._kili_api_gateway.create_project(
+                input_type=input_type,
+                json_interface=project_copied["jsonInterface"],
+                title=title,
+                description=description,
+                project_type=project_type,
+                compliance_tags=compliance_tags,
+            )
+            if project_copied["instructions"]:
+                self.update_properties_in_project(
+                    project_id=new_project_id,
+                    can_navigate_between_assets=None,
```

Comment:
Do we need to specify the None parameters, or can we ommit them ?

Reply:
I didn’t manage to ommit it but maybe you have a better solution ?

Reply:
for my understandig they are all defined as Optional, so you should be able to ignore all Optional one.

What error do you have when you omit them ?

Reply:
If I remove them i have this error
`Arguments missing for parameters "project_id", "can_navigate_between_assets", "can_skip_asset", "compliance_tags", "consensus_mark", "consensus_tot_coverage", "description", "honeypot_mark", "input_type", "json_interface", "min_consensus_size", "number_of_assets", "number_of_skipped_assets", "number_of_remaining_assets", "number_of_reviewed_assets", "review_coverage", "should_relaunch_kpi_computation", "title", "use_honeypot", "metadata_types"`
They are mandatory in the method `update_properties_in_project` in the same file, even if I use `self._kili_api_gateway.update_properties_in_project` I have the same error

Reply:
Yes, we need to specify default value for parameters to be optional, can we add those ?🙏

Reply:
Yes ok, you’re right that's better, i've update the code